language/documentation/book/src/generics.md

```diff
@@ -135,6 +135,8 @@ module M {
 
 ## Unused Type Parameters
 
+An unused type parameter is one that doesn't show up at runtime,
```

Comment:
I would reword this a bit "does not appear in a field" or something similar, as it does appear "at runtime" in the VM